examples/step-21/doc/intro.dox

```diff
@@ -108,7 +108,7 @@ In summary, what we get are the following two equations:
   - \nabla \cdot (\mathbf{K}\lambda(S) \nabla p) &=& q
   \qquad \textrm{in}\ \Omega\times[0,T],
   \\
-  S_{t} + \mathbf{u} \cdot \nabla F(S) &=& 0
+  S_{t} + \nabla (F(S) \cdot \mathbf{u}) &=& 0
```

Comment:
Well, but that's not quite the same, is it?

Also, can you move the `\cdot` in the formula to after the `\nabla`?